src/phone_sms_terms.js

```diff
@@ -20,6 +20,9 @@
 
 var SMS_SUBSCRIBE_DIV = '#id_subscribe_sms';
 
+var params = new URLSearchParams(window.location.search);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#Browser_compatibility says this doesn't work in IE at all. I'm not sure if we have a defined list of browsers we support, but I think we probably need to support some versions of IE.

https://stackoverflow.com/questions/2090551/parse-query-string-in-javascript#2091331 looks like an IE-friendly way to parse the query string.